src/controller/CHIPDeviceController.cpp

```diff
@@ -1655,17 +1727,56 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
 
     switch (step)
     {
+    case CommissioningStage::kReadVendorId: {
```

Comment:
It might be worth reading all of the Basic bits we care about plus the network commissioning feature maps in s single read, using the same setup as #14410 is doing.  Just instead of a single wildcard path, pass in whatever list paths we care about, some of which might be wildcards...  Then wait for the read cache to get back to us, and get the values from that.  It'd probably cut down significantly on the round-trips.

Reply:
I agree, but I got that network cluster stuff working at 11:00 friday night...sliding in under the deadline like Indiana jones (if we stretch a bit what it means to hit the deadlines). I agree that would be preferable, but at this point I think I would need approval to put it in.

Reply:
I think my point is that once #14410 merges, we should consider using its machinery here to just get all that data in a single read....  

That does not need to block this PR, but can happen as a folowup.  And we can figure out the process as needed, I expect.

Reply:
sounds good. https://github.com/project-chip/connectedhomeip/issues/14669 to track